modules/rest-api/pages/rest-initialize-cluster.adoc

```diff
@@ -22,7 +22,7 @@ xref:rest-api:rest-initialize-node.adoc[Initializing a Node],
 xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials],
 xref:rest-api:rest-name-node.adoc[Naming a Node],
 xref:rest-api:rest-configure-memory.adoc[Configuring Memory],
-xref:rest-api:rest-set-up-services.adoc[Assigning Services],
+xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Node],
```

Comment:
`xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Single Node],`